pkg-manager/plugin-commands-installation/test/update/interactive.ts

```diff
@@ -311,27 +311,25 @@ test('interactively update should ignore dependencies from the ignoreDependencie
     storeDir,
   })
 
-  expect(prompt.mock.calls[0][0].choices).toStrictEqual(
-    [
-      {
-        choices: [
-          {
-            disabled: true,
-            hint: '',
-            name: 'Package                                                    Current   Target            URL ',
-            value: '',
-          },
-          {
-            message: chalk`micromatch                                                   3.0.0 ❯ 3.{yellowBright.bold 1.10}                `,
-            value: 'micromatch',
-            name: 'micromatch',
-          },
-        ],
-        name: '[dependencies]',
-        message: 'dependencies',
-      },
-    ]
-  )
+  expect(prompt.mock.calls[0][0].choices).toStrictEqual([
+    {
+      choices: [
+        {
+          disabled: true,
+          hint: '',
+          name: 'Package                                                    Current   Target            URL ',
+          value: '',
+        },
+        {
+          message: chalk`micromatch                                                   3.0.0 ❯ 3.{yellowBright.bold 1.10}                `,
+          value: 'micromatch',
+          name: 'micromatch',
+        },
+      ],
+      name: '[dependencies]',
+      message: 'dependencies',
+    },
+  ])
```

Comment:
Looks like here only the formatting changed. Revert it back.

Reply:
done

Reply:
is seems the semicolon not write by me, just run `pnpm run lint --fix`, it will change here automatally.